deploy/kubernetes/examples/demo-secret-configuration-file.json

```diff
@@ -0,0 +1,24 @@
+{
+  "demo-system_id1": {
+    "username": "demo-username",
+    "password": "demo-password",
+    "management_address": "demo-management-address",
+    "supported_topologies": [
+      {
+        "topology.block.csi.ibm.com/zone": "demo-zone",
+        "topology.block.csi.ibm.com/dev": "demo-dev"

```

Comment:
I think the `region` is more common than `dev`

Reply:
right, that why I wrote dev (which is common in other docs as well), to emphasize that we support more than the common cases.

Reply:
then how about `.../demo-region` and `.../demo-zone`?

Reply:
how about more general? like `demo-suffix1` and `demo-suffix2`

Reply:
ok, done

Reply:
I [meant](https://github.com/IBM/ibm-block-csi-driver/pull/353#discussion_r682355801) that the keys suffix should contain `demo-` as well, to show that it's flexible as well

Reply:
done